### PR TITLE
40 sprint sailor update 267

### PR DESCRIPTION
**Pull request by kirill-levitskiy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,28 +14,28 @@
...
+    "elasticio-sailor-nodejs": "2.6.7",
     "micromatch": "4.0.2",
-    "moment": "2.24.0",
-    "ssh2-sftp-client": "4.3.0"
+    "moment": "2.25.3",
+    "ssh2-sftp-client": "5.1.2"
   },
   "engines": {
-    "node": ">=10"
+    "node": "12"
```
Comment (denyshld) [inline package.json:27]: '>=12'

File: package.json [hunk 2]
```diff
@@ -14,28 +14,28 @@
   "author": "elastic.io GmbH",
   "license": "BSD-2-Clause",
   "dependencies": {
-    "@elastic.io/component-commons-library": "0.0.6",
-    "@elastic.io/oih-standard-library": "1.0.1",
+    "@elastic.io/component-commons-library": "1.1.2",
+    "@elastic.io/oih-standard-library": "1.0.3",
```
Comment (shulkaolka) [inline package.json:18]: are you sure that all library changes should be used in this component?

File: package.json (hunk 1 above)
Comment (kirill-levitskiy) [reply package.json]: Fixed

File: package.json (hunk 2 above)
Comment (kirill-levitskiy) [reply package.json]: Changes were reverted

**Comment by kirill-levitskiy** [comment]
> circle ci build failed, please, check

Known problem with outdated credentials in the circleci
We can't fix it

**Comment by kirill-levitskiy** [comment]
Fixed circleci

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
+## 1.2.2 (May 9, 2020)
```
Comment (uaArsen) [inline CHANGELOG.md:1]:
```suggestion
## 1.2.2 (May 19, 2020)
```

**Pull request merged**